### PR TITLE
Add GitHub action for terraform fmt and validation

### PR DESCRIPTION
**Pull request by johscheuer**

### CODE REVIEW

File: .github/workflows/terraform.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  terraform:
+    name: 'Terraform'
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout'
+        uses: actions/checkout@v2
+      - name: 'Terraform Format'
+        uses: hashicorp/terraform-github-actions@v0.6.3
+        with:
+          tf_actions_version: 0.12.13
```
Comment (hikhvar) [inline .github/workflows/terraform.yml]: I think we should update to 0.12.18. That is the currently most recent terraform version

File: .github/workflows/terraform.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+          tf_actions_subcommand: 'fmt'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: 'Terraform Init'
+        uses: hashicorp/terraform-github-actions@v0.6.3
+        with:
+          tf_actions_version: 0.12.13
```
Comment (hikhvar) [inline .github/workflows/terraform.yml]: Here too.

File: .github/workflows/terraform.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+          tf_actions_subcommand: 'init'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Create dummy account.json file
+        run: touch account.json
+      - name: 'Terraform Validate'
+        uses: hashicorp/terraform-github-actions@v0.6.3
+        with:
+          tf_actions_version: 0.12.13
```
Comment (hikhvar) [inline .github/workflows/terraform.yml]: And here.

File: .github/workflows/terraform.yml (hunk 1 above)
Comment (johscheuer) [reply .github/workflows/terraform.yml]: Good point I updated the version

**Comment by johscheuer** [comment]
`- Your workflow file was invalid: The pipeline is not valid. .github/workflows/terraform.yml: Anchors are not currently supported. Remove the anchor 'terraform_github_action'`

https://giphy.com/gifs/sad-please-choked-up-ZEgBHVeRlmTqjCPlqx/fullscreen

**Comment by hikhvar** [comment]
LGTM

**Pull request merged**